pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -221,21 +223,35 @@ process buscoAnnot {
     output:
         tuple path("annotation.gtf"), path(genome), emit: busco_annot
     script:
+    if (params.use_anno == "")
+    """
+        SENS=""
+        if [ "${params.sensitive}" != "" ];
+        then
+            SENS="-M0 -k5"
+        fi
+        ${params.miniprot_exe} \$SENS -t ${params.cores} -d genome.mpi $genome
+        ${params.miniprot_exe} -N 0 -Iu -t ${params.cores} --gff genome.mpi $busco_prot | grep -v '##PAF' \
+        | awk '/mRNA/{split(\$9,a,/ID=|;/); split(\$9,b,/Target=|\s/)}{sub(a[2], b[2]); print}' > annotation.gtf
```

Comment:
```suggestion
        | awk -F "\t" '$3=="mRNA" {match($9, /Target=([^; ]+)/, m); sub(/ID=[^; ]+/, sprintf("ID=%s", m[1]), $9); print}' > annotation.gtf
```
This includes several suggested changes:
- setting an explicit tab delimiter (`-F "\t"`) — to avoid whitespace issues;
- replacing the feature-type column regex with an exact string match `$3=="mRNA"` — to avoid using a regex where possible;
- replacing the `split` function calls with a `match` of the `Target` attribute and a `sub` of the matched target ID into the `ID` attribute — to make the intent clearer.

Reply:
Whoops! 🤦 I've now put the `match` and `sub` calls into separate blocks so that the substitution would be done on every line, and not just the `mRNA` line.